cartographer/io/proto_stream_deserializer.cc

```diff
@@ -57,17 +57,17 @@ ProtoStreamDeserializer::ProtoStreamDeserializer(
          "`SerializationHeader`, but got field tag "
       << pose_graph_.data_case();
 
-  CHECK(ReadNextSerializedData(&all_trajectory_builder_options_))
-      << "Serialized stream misses `AllTrajectoryBuilderOptions`.";
-  CHECK(all_trajectory_builder_options_.has_all_trajectory_builder_options())
+  CHECK(ReadNextSerializedData(&builder_options_))
```

Comment:
Does this break compatibility?

Reply:
It is the same proto, just renamed to builder options (instead of all trajectory builder options), since it now contains the map builder options as well.

Reply:
Oh yes, true!